src/Model/Subsite.php

```diff
@@ -919,29 +918,6 @@ public function activate()
         Subsite::changeSubsite($this);
     }
 
-    /**
-     * @param array $permissionCodes
-     * @return DataList<Member>
-     */
-    public function getMembersByPermission($permissionCodes = ['ADMIN'])
```

Comment:
This method was added in https://github.com/silverstripe/silverstripe-subsites/commit/5e9b67216f1191c0e19f5dda75ba6e685cd5335f which has no context whatsoever about what its intended use was or how it was intended to work.

Right now it just simply doesn't work. I had a choice between fixing it and removing it, and since there's no context about how it _should_ work and it's impossible for anyone to use right now anyway, fixing it just didn't seem worthwhile.